slo/promql_test.go

```diff
@@ -1086,3 +1099,24 @@ func TestReplacer(t *testing.T) {
 		})
 	}
 }
+
+func TestLatencyNativeBurnrateGrouping(t *testing.T) {
+	objective := objectiveHTTPNativeLatencyGrouping()
+
+	burnrateQuery := objective.Burnrate(5 * time.Minute)
+
+	require.True(t,
+		strings.Contains(burnrateQuery, "sum by (handler, job)") || strings.Contains(burnrateQuery, "sum by (job, handler)"),
```

Comment:
If this isn't sorted and stable, we should make it so. We can do that in a follow-up up though.